ext/message_admin/CRM/MessageAdmin/Settings.php

```diff
@@ -9,8 +9,15 @@ public static function getAll() {
       ->addSelect('name', 'label')
       ->addOrderBy('label')
       ->execute();
+    $allLangsIdx = array_combine($allLangs->column('name'), $allLangs->column('label'));
```

Comment:
I have no idea what this variable name is supposed to mean - non-blocking but confusing